src/components/ErrorBoundary/withErrorBoundary.js

```diff
@@ -1,17 +1,17 @@
 /**
  * @flow
- * @file withErrorBoundary HOC
+ * @file withErrorBoundary HOC which adds error boundaries as well as error error logging
  * @author Box
  */
 
 import * as React from 'react';
 import ErrorBoundary from './ErrorBoundary';
 
-const withErrorBoundary = (WrappedComponent: React.ComponentType<any>) =>
+const withErrorBoundary = (errorType: ErrorTypes) => (WrappedComponent: React.ComponentType<any>) =>
```

Comment:
Minor, probably the type should also be singular `ErrorType` because reading it I thought it was array of errors